react/features/toolbox/middleware.js

```diff
@@ -7,6 +7,48 @@ import {
     SET_TOOLBOX_TIMEOUT
 } from './actionTypes';
 
+import {
+    SET_VIDEO_AVAILABLE,
+    SET_VIDEO_MUTED
+} from '../../features/base/media/actionTypes';
+
+import {
+    setToolbarButton
+} from './actions.native.js';
```

Comment:
Without thinking too deeply about what would be better as a general rule, I'd go with `'./actions'` specifically for consistency with how the other import statements are written in the same file.